docs-source/content/release-notes.md

```diff
@@ -4,7 +4,7 @@ date: 2019-03-15T11:25:28-04:00
 draft: false
 ---
 
-### Recent changes
+### Releases
 
 | Date | Version | Introduces backward incompatibilities? | Changes |
```

Comment:
Changes -> Change (I let this go before but it was a mistake. It should be singular, change, just like all the other headings are singular, Date, Version, Issue, and Description. The question mark after Introduces backward incompatibilities? is ok.) I looked at the Change log in the Hugo UI and it all looks fine, definitely good to go.